vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -773,8 +761,7 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           }
           let cacheToClear = grid._cache;
           if (parentKey) {
-            const cacheAndIndex = grid._cache.getCacheAndIndexByKey(pkey);
-            cacheToClear = cacheAndIndex.cache.itemCaches[cacheAndIndex.scaledIndex];
+            cacheToClear = grid._cache.getCacheByKey(pkey);
```

Comment:
Would be ideal to have wtr test coverage for all refactored connector code just to be on the safe side. Many places in the connector, such as this, haven't yet been covered. This particular change is not too complex though so it's not an issue. Much of the code is also covered by ITs but it's impossible to tell exactly which parts.